scss/navbar.scss

```diff
@@ -0,0 +1,30 @@
+@import 'variables';
+
+.navbar {
+  position: fixed;
+  top: 0;
+  width: 100%;
+  position: -webkit-sticky;
+  position: sticky;
+  z-index: 200;
+  background: white;
+  height: $nav-height;
+  box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
+
+}
+
+.nav-img { 
+  left: 20px;
```

Comment:
I would use margin instead, why dont you position navbar elements with flex on  `nav` element?